pkgs/servers/passopolis/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, antBuild, fetchgit, git, python }:
+
+let
+  version = "unstable-2016-05-07";
```

Comment:
I have suggested to add `ùnstable` because of https://nixos.org/nixpkgs/manual/#sec-package-naming